PGDATA is unknown if use .walg.json

```
cat $HOME/.walg.json
{
  "WALE_S3_PREFIX": "s3://pg-backups",
  "AWS_ACCESS_KEY_ID": "minio",
  "AWS_ENDPOINT": "http://172.26.9.241:9000",
  "AWS_S3_FORCE_PATH_STYLE": "true",
  "AWS_SECRET_ACCESS_KEY": "miniosecret",
  "PGDATA": "/var/lib/pgsql/9.6/data/",
  "PGHOST": "/var/run/postgresql/.s.PGSQL.5432",
  "WALG_UPLOAD_CONCURRENCY": "2",
  "WALG_DOWNLOAD_CONCURRENCY": "2",
  "WALG_UPLOAD_DISK_CONCURRENCY": "2",
  "WALG_DELTA_MAX_STEPS": "7",
  "WALG_COMPRESSION_METHOD": "brotli"
}
```

Get error:
```
usr/local/bin/wal-g backup-list
WARNING: 2020/02/02 12:33:32.793829 PGDATA is unknown
WARNING: 2020/02/02 12:33:32.793865 We found that some variables in your config file detected as 'Unknown'. 
  If this is not right, please create issue https://github.com/wal-g/wal-g/issues/new
name                          last_modified        wal_segment_backup_start
```

```
+++ b/internal/config.go
@@ -37,6 +37,7 @@ const (
        PgPortSetting                = "PGPORT"
        PgUserSetting                = "PGUSER"
        PgHostSetting                = "PGHOST"
+       PgDataSetting                = "PGDATA"
        PgPasswordSetting            = "PGPASSWORD"
        PgDatabaseSetting            = "PGDATABASE"
        PgSslModeSetting             = "PGSSLMODE"
@@ -95,6 +96,7 @@ var (
                PgPortSetting:     true,
                PgUserSetting:     true,
                PgHostSetting:     true,
+               PgDataSetting:     true,
                PgPasswordSetting: true,
                PgDatabaseSetting: true,
                PgSslModeSetting:  true,
```

Error:
```
make pg_build
(cd main/pg && go build -mod vendor -tags "brotli lzo""" -o wal-g -ldflags "-s -w -X github.com/wal-g/wal-g/cmd/pg.BuildDate=`date -u +%Y.%m.%d_%H:%M:%S` -X github.com/wal-g/wal-g/cmd/pg.GitRevision=`git rev-parse --short HEAD` -X github.com/wal-g/wal-g/cmd/pg.WalgVersion=`git tag -l --points-at HEAD`")
# github.com/wal-g/wal-g/internal
../../internal/config.go:40:2: PgDataSetting redeclared in this block
	previous declaration at ../../internal/config.go:35:33
Makefile:29: recipe for target 'pg_build' failed
make: *** [pg_build] Error 2
```

--- a/internal/config.go
+++ b/internal/config.go
@@ -95,6 +95,7 @@ var (
 		PgPortSetting:     true,
 		PgUserSetting:     true,
 		PgHostSetting:     true,
+                PgDataSetting:     true,
 		PgPasswordSetting: true,
 		PgDatabaseSetting: true,
 		PgSslModeSetting:  true,

--- a/tests_func/helpers/walg.go
+++ b/tests_func/helpers/walg.go
@@ -103,6 +103,9 @@ func (w *WalgUtil) runCmd(run []string) (ExecResult, error) {
 
 func (w *WalgUtil) PushBackup() (string, error) {
 	PgDataSettingString, ok := internal.GetSetting(internal.PgDataSetting)
+	if ok == false {
+		tracelog.InfoLogger.Print("\nPGDATA is not set in the conf.\n")
+	}
 	if w.cliPath != PgDataSettingString {
 		tracelog.WarningLogger.Printf("cliPath '%s' differ from conf PGDATA '%s'\n", w.cliPath, PgDataSettingString)
 	}

--- a/tests_func/helpers/walg.go
+++ b/tests_func/helpers/walg.go
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal"
 )
 
 type Sentinel struct {
@@ -101,7 +102,7 @@ func (w *WalgUtil) runCmd(run []string) (ExecResult, error) {
 }
 
 func (w *WalgUtil) PushBackup() (string, error) {
-	PgDataSettingString, ok := GetSetting(PgDataSetting)
+	PgDataSettingString, ok := internal.GetSetting(internal.PgDataSetting)
 	if w.cliPath != PgDataSettingString {
 		tracelog.WarningLogger.Printf("cliPath '%s' differ from conf PGDATA '%s'\n", w.cliPath, PgDataSettingString)
 	}

--- a/tests_func/helpers/walg.go
+++ b/tests_func/helpers/walg.go
@@ -101,6 +101,10 @@ func (w *WalgUtil) runCmd(run []string) (ExecResult, error) {
 }
 
 func (w *WalgUtil) PushBackup() (string, error) {
+	PgDataSettingString, ok := GetSetting(PgDataSetting)
+	if w.cliPath != PgDataSettingString {
+		tracelog.WarningLogger.Printf("cliPath '%s' differ from conf PGDATA '%s'\n", w.cliPath, PgDataSettingString)
+	}
 	exec, err := w.runCmd([]string{"backup-push"})
 	if err != nil {
 		return "", err

